js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -0,0 +1,16 @@
+
+function leastCommonMultiple(...args) {
+  let largestNumber = Math.max(args[0],args[1])
```

Comment:
Always try to avoid usage of args inside the program. Either extract the args to some variables and then use them.